pom.xml

```diff
@@ -387,11 +387,17 @@
                         </execution>
                     </executions>
                 </plugin>
+                <plugin>
+                    <groupId>org.scalatest</groupId>
+                    <artifactId>scalatest-maven-plugin</artifactId>
+                    <version>${scalatest.maven.version}</version>
+                </plugin>
             </plugins>
         </pluginManagement>
         <plugins>
             <!-- disable surefire -->
             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Let's remove the surefire plugin completely.

Reply:
Surefire plugin fully removed in plugin ce7c3fa686ae91c964dc13a408e608ae1c5f17ee.